tests/dogtag/pytest-ansible/installation/roles/Test_Trigger/tasks/configure_ca.yml

```diff
@@ -23,3 +23,33 @@
   - /tmp/test_dir/ca.cfg
   - /tmp/test_dir/constants.py
 
+- name: Configure LDAPS in CA config
+  block:
+  - name: Add ldaps port in the CA file
+    lineinfile: dest={{item}} insertafter="^pki_ds_ldap_port" line="pki_ds_ldaps_port = ldapSecureServerPort"
```

Comment:
suggestion: Use `jinja2` templating when dealing with replacing variables inside ansible

Ref: https://docs.ansible.com/ansible/latest/user_guide/playbooks_templating.html

Reply:
We are just pushing the existing downstream code, we can follow the jinja tempering once we are done syncing the upstream and downstream code.
Currently i have excluded the playbook changes for another MR. So marking this comment resolved for this MR.
This MR consists changes related to executing existing tests in Fedora32.